spec/support/features/document_capture_step_helper.rb

```diff
@@ -0,0 +1,67 @@
+module DocumentCaptureStepHelper
+  def attach_and_submit_images
+    attach_images
+
+    # If selfie is required, we simulate a submission with the document capture
+    # react component. As a result, the page has already been submitted so
+    # submitting is a noop
+
+    if javascript_enabled? && !selfie_required?
+      click_on 'Submit'
+      # Wait for the background image job to finish and success flash to appear before continuing
+      Capybara.using_wait_time(3) do
+        expect(page).to have_content(t('doc_auth.headings.capture_complete'))
+      end
+    elsif !javascript_enabled?
+      click_idv_continue
+    end
+  end
+
+  def attach_images
+    if javascript_enabled?
+      attach_images_with_js
+    else
+      attach_images_without_js
+    end
+  end
+
+  def attach_images_with_js
+    if selfie_required?
+      simulate_image_upload_api_submission
+    else
+      attach_file 'Front of your ID', 'app/assets/images/logo.png'
+      attach_file 'Back of your ID', 'app/assets/images/logo.png'
+    end
+  end
+
+  def attach_images_without_js
+    attach_file 'doc_auth_front_image', 'app/assets/images/logo.png'
+    attach_file 'doc_auth_back_image', 'app/assets/images/logo.png'
+    attach_file 'doc_auth_selfie_image', 'app/assets/images/logo.png' if selfie_required?
+  end
+
+  def simulate_image_upload_api_submission
```

Comment:
I'm guessing this won't work as expected with `doc_auth_enable_presigned_s3_urls` enabled?

Reply:
It will not. I'm cooking up another PR that re-arranges some of the document capture step specs and adds more coverage when that's enabled. Pulling together something similar for that method there.